src/tasks/minions/minigames/titheFarmActivity.ts

```diff
@@ -0,0 +1,132 @@
+import { Task, KlasaMessage } from 'klasa';
+
+import { TitheFarmActivityTaskOptions } from '../../../lib/types/minions';
+import { channelIsSendable } from '../../../lib/util/channelIsSendable';
+import { saidYes, noOp, roll } from '../../../lib/util';
+import itemID from '../../../lib/util/itemID';
+import { UserSettings } from '../../../lib/settings/types/UserSettings';
+import { Time, Events, Emoji } from '../../../lib/constants';
+import { SkillsEnum } from '../../../lib/skilling/types';
+import bankHasItem from '../../../lib/util/bankHasItem';
+import getUsersPerkTier from '../../../lib/util/getUsersPerkTier';
+
+export default class extends Task {
+	async run({ userID, channelID }: TitheFarmActivityTaskOptions) {
+		const baseHarvest = 85;
+		let lootStr = '';
+
+		const user = await this.client.users.fetch(userID);
+		const channel = await this.client.channels.fetch(channelID).catch(noOp);
+
+		const farmingLvl = user.skillLevel(SkillsEnum.Farming);
+		const titheFarmsCompleted = user.settings.get(UserSettings.Stats.TitheFarmsCompleted);
+		const titheFarmPoints = user.settings.get(UserSettings.Stats.TitheFarmPoints);
+		const determineHarvest = baseHarvest + Math.min(15, titheFarmsCompleted);
+		const determinePoints = determineHarvest - 74;
+
+		await user.settings.update(UserSettings.Stats.TitheFarmsCompleted, titheFarmsCompleted + 1);
```

Comment:
Is there any easy way of doing it to where we can pass 2 different updates in the same user.settings.update()? 

The other obvious route is to just make a titheFarm object and store both properties on there and then update them in 1 call as an object with modified property values but I was just looking to avoid it if there's an easier way.